src/Groups/homomorphisms.jl

```diff
@@ -1,10 +1,27 @@
+function Base.show(io::IO, ::MIME"text/plain", x::GAPGroupHomomorphism)
+   println(io, "Group homomorphism")
+   io = AbstractAlgebra.pretty(io)
+   print(io, AbstractAlgebra.Indent())
```

Comment:
```suggestion
   io = pretty(io)
   print(io, Indent())
```
all of this gets imported in `src/imports.jl`